app/src/main/cpp/src/components/transport_manager/include/transport_manager/bluetooth_le/bluetooth_le_transport_adapter.h

```diff
@@ -86,9 +87,11 @@ class BluetoothLeTransportAdapter : public TransportAdapterImpl {
 
   private:
     BleClient ble_client_;
-    BleServer ble_server_;
+    std::unique_ptr<BleServer> ble_server_;
```

Comment:
Why we need pointers here? Is some profit there?

Reply:
ptr removed in 66d56ca414ac2d17dea823a2186a8d3f2569ec83